firstvoices/backend/tests/test_apis/test_dictionary_api.py

```diff
@@ -358,6 +358,20 @@ def assert_update_patch_response(self, original_instance, data, actual_response)
             },
         ]
 
+    @pytest.mark.skip(
```

Comment:
The only optional text fields on entries are within arrayFields, so a null value would be [] rather than None which we already handle.